apps/client/src/features/event-editor/EventEditor.tsx

```diff
@@ -16,23 +16,24 @@ export type EventEditorSubmitActions = keyof OntimeEvent;
 export type EditorUpdateFields = 'cue' | 'title' | 'presenter' | 'subtitle' | 'note' | 'colour';
 
 export default function EventEditor() {
-  const openId = useAppMode((state) => state.editId);
+  const { eventsToEdit } = useEventSelection();
```

Comment:
is it not preferrable using the memoised version as in the code that existed?

Reply:
From the [docs](https://github.com/pmndrs/zustand#selecting-multiple-state-slices) I'm reading, it doesn't make a difference if it's an object since it's using a strict equality check. Am I getting this right?

Reply:
I dont think you are getting it right.
The memoised selector will give you a dataset which is "protected" from changes to the rest of the stored object

I made a [simple sandbox to demonstrate](https://codesandbox.io/p/sandbox/test-zustand-selector-f9ng9z?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clqdhhk4f00063b6lza8wgjyl%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clqdhhk4f00023b6lly9cntjv%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clqdhhk4f00033b6lfjd37f6w%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clqdhhk4f00053b6l80g1f7ko%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clqdhhk4f00023b6lly9cntjv%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clqdhhk4f00013b6lzay86o3s%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fsrc%252Findex.js%2522%257D%255D%252C%2522id%2522%253A%2522clqdhhk4f00023b6lly9cntjv%2522%252C%2522activeTabId%2522%253A%2522clqdhhk4f00013b6lzay86o3s%2522%257D%252C%2522clqdhhk4f00053b6l80g1f7ko%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clqdhhk4f00043b6lh5ey6ogg%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522id%2522%253A%2522clqdhhk4f00053b6l80g1f7ko%2522%252C%2522activeTabId%2522%253A%2522clqdhhk4f00043b6lh5ey6ogg%2522%257D%252C%2522clqdhhk4f00033b6lfjd37f6w%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clqdhhk4f00033b6lfjd37f6w%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)

Reply:
Thanks for the impressive demo! Makes a lot of sense

Resolved with https://github.com/cpvalente/ontime/pull/497/commits/da1b32811475028b5d8d42323e9ad678255913a1